stdlib/_socket.pyi

```diff
@@ -527,6 +527,7 @@ class socket:
     family: int
     type: int
     proto: int
+    timeout: float | None
```

Comment:
This should be a property without a setter, because it cannot be assigned to (you have to use `settimeout()`).